--- /app/qlkt/Views/RewardListView.xaml
+++ /app/qlkt/Views/RewardListView.xaml
@@ -172,7 +172,7 @@
         <!-- DATA GRID -->
         <Border Grid.Row="2" CornerRadius="8" BorderBrush="#E0E0E0" BorderThickness="1" Background="White" ClipToBounds="True">
             <DataGrid x:Name="dgRewardList" BorderThickness="0">
                 <DataGrid.Columns>
-                    <DataGridTextColumn Header="" Binding="{Binding STT}" Width="60" ElementStyle="{StaticResource {x:Type TextBlock}}">
+                    <DataGridTextColumn Header="" Binding="{Binding STT}" Width="60">
                         <DataGridTextColumn.HeaderStyle>
                             <Style TargetType="DataGridColumnHeader" BasedOn="{StaticResource {x:Type DataGridColumnHeader}}">
                                 <Setter Property="HorizontalContentAlignment" Value="Center"/>
